### PR TITLE
Feature/rt fields

### PR DESCRIPTION
**Pull request by Dalamar42**
This PR is to ensure that the Broadcast#newEpisode field gets ingested from Owl and appears in the API.

Please review commit by commit:

1. Cleanup as per code style -> Please review
2. Automatic re-order of code as per code style -> IntelliJ did this. A cursory look is enough
3. Code style; guava function removal and method ordering  -> A tiny bit I missed in 1. Please review
4. Import Broadcast#newEpisode field from Owl and add to API -> The actual feature
5. Clean up OutputContentMerger according to code style -> Mostly IntelliJ refactors. A cursory look please

### CODE REVIEW

File: atlas-api/src/main/java/org/atlasapi/output/writers/BroadcastWriter.java [hunk 1]
```diff
@@ -31,17 +31,25 @@
...
     private final ChannelResolver channelResolver;
+    private final ChannelWriter channelWriter;
+
+    public BroadcastWriter(
+            String listName,
+            NumberToShortStringCodec codec,
+            ChannelResolver channelResolver,
+            ChannelGroupResolver channelGroupResolver
+    ) {
+        this.aliasWriter = new AliasWriter();
```
Comment (emilssolmanis) [inline atlas-api/src/main/java/org/atlasapi/output/writers/BroadcastWriter.java]: Why pull these in here?

File: atlas-api/src/main/java/org/atlasapi/output/writers/BroadcastWriter.java [hunk 2]
```diff
@@ -69,17 +86,17 @@ public void write(Broadcast entity, FieldWriter writer, OutputContext ctxt) thro
...
-        if (entity.getChannelId() != null) {
-            Optional<Channel> maybeChannel = Futures.get(channelResolver.resolveIds(ImmutableList.of(
-                    entity.getChannelId())), IOException.class)
-                    .getResources().first();
-            if (maybeChannel.isPresent()) {
-                channel = maybeChannel.get();
-            } else {
-                log.error("Unable to resolve channel {}", entity.getChannelId());
-            }
+        Optional<Channel> maybeChannel = Futures.get(channelResolver.resolveIds(ImmutableList.of(
```
Comment (emilssolmanis) [inline atlas-api/src/main/java/org/atlasapi/output/writers/BroadcastWriter.java]: This is still as fucked up, throw a ton of newlines at this and make it style-guide compliant (and therefore readable). Stuff like
```
Futures.get(
    channelResolver.resolveIds(
        ImmutableList.of(
```

File: atlas-api/src/main/java/org/atlasapi/output/writers/BroadcastWriter.java (hunk 1 above)
Comment (Dalamar42) [inline atlas-api/src/main/java/org/atlasapi/output/writers/BroadcastWriter.java]: Consistency? That's how we do it elsewhere and it's nice to see how all dependencies are initialised in one place

File: atlas-api/src/main/java/org/atlasapi/output/writers/BroadcastWriter.java (hunk 2 above)
Comment (Dalamar42) [inline atlas-api/src/main/java/org/atlasapi/output/writers/BroadcastWriter.java]: ack

File: atlas-api/src/main/java/org/atlasapi/output/writers/BroadcastWriter.java (hunk 1 above)
Comment (emilssolmanis) [inline atlas-api/src/main/java/org/atlasapi/output/writers/BroadcastWriter.java]: Disagree, but fine. IMO it's more obvious that these are completely independent of anything if they're initialised in the way they were done before, but I don't care enough.

**Comment by emilssolmanis** [comment]
I've no idea wth a "new episode" is, but technically lgtm. :shipit:

File: atlas-api/src/main/java/org/atlasapi/output/writers/BroadcastWriter.java (hunk 1 above)
Comment (Dalamar42) [inline atlas-api/src/main/java/org/atlasapi/output/writers/BroadcastWriter.java]: The other reason I like this approach is that these things should be yank'able to allow tests to mock them. The fact we are not doing it here is another story.

**Pull request merged**